### PR TITLE
fix tests so they run against the correct stage

### PR DESCRIPTION
**Pull request by philvarner**

### CODE REVIEW

File: tests/test-query.ipynb [hunk 1]
```diff
@@ -38,50 +38,35 @@
...
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "response = maap.executeQuery(\n",
-    "     # dictionary-like object specifying the dataset to query\n",
-    "    src={\n",
+    "if stage != \"dit\": # SKIP until dit is fixed\n",
```
Comment (tracetechnical) [inline tests/test-query.ipynb:41]: This doesn't appear to be in a comment block? Unless I have misunderstood the readme

**Review by tracetechnical — CHANGES_REQUESTED** [review]
One small Q around a Skipped DIT test

**Comment by tracetechnical** [comment]
It would be amazing if there were a way to .gitignore or similar execution count changes. Unless there is some majorly legit reason they get commited to git.

File: tests/test-query.ipynb (hunk 1 above)
Comment (philvarner) [reply tests/test-query.ipynb:41]: The intention here is that when the deployed services for a stage are broken wrt a test, that we skip running those tests against that stage. Here, Query services are broken for dit, so these tests are ONLY run in production (and presumably staging when we actually have that deployed as a stage)

**Comment by philvarner** [comment]
> It would be amazing if there were a way to .gitignore or similar execution count changes. Unless there is some majorly legit reason they get commited to git.

One way might be to have a pre-commit check that set them all to null (or at least checked that they were all still null), like you'd usually do with a linter.

File: tests/test-query.ipynb (hunk 1 above)
Comment (tracetechnical) [reply tests/test-query.ipynb:41]: Doy! Apologies. More caffeine required before I comment next time.

**Comment by tracetechnical** [comment]
> > It would be amazing if there were a way to .gitignore or similar execution count changes. Unless there is some majorly legit reason they get commited to git.
> 
> One way might be to have a pre-commit check that set them all to null (or at least checked that they were all still null), like you'd usually do with a linter.

From a reviewers point of view, it would decrease the cognitive load a bit, and avoid line fatigue risking stuff getting missed. So would be a nice to have, but by no means essential.

**Comment by philvarner** [comment]
I filed https://trello.com/c/LKMoPcTK/328-tests-pre-commit-action-check-that-all-executioncount-values-are-null about this.

**Pull request merged**